### ISSUE TITLE
Champion unable to select Pharasma as a Diety. 

### ISSUE THREAD

**Issue by Grizzally** [MrPrimate/pathmuncher#49]
**Describe the bug**

**To Reproduce**
1. Go to Import
2. Click on Import
3. Make sure you are importing a Champion. 

**Expected behavior**
I expected to be able to select Pharasma and her Font's and a choice of her Domain.

**Screenshots**
![image](https://github.com/MrPrimate/pathmuncher/assets/174618838/c9f2f114-2e7c-4481-8227-c2a7607ac1b3)

 - In Pathbuilder please got to Export -> Unfortunately I don't have the link, it's my Players Character. I have the JSON. 
 - 
{"success":true,"build":{"name":"Bulwark","class":"Champion","dualClass":null,"level":4,"ancestry":"Orc","heritage":"Hold-Scarred Orc","background":"Warrior","alignment":"N","gender":"Male","age":"26","deity":"Not set","size":2,"sizeName":"Medium","keyability":"str","languages":["Common","Jotun"],"rituals":[],"resistances":[],"inventorMods":[],"attributes":{"ancestryhp":10,"classhp":10,"bonushp":2,"bonushpPerLevel":1,"speed":25,"speedBonus":-5},"abilities":{"str":18,"dex":14,"con":14,"int":10,"wis":10,"cha":12,"breakdown":{"ancestryFree":["Str","Dex"],"ancestryBoosts":[],"ancestryFlaws":[],"backgroundBoosts":["Con","Str"],"classBoosts":["Str"],"mapLevelledBoosts":{"1":["Str","Con","Dex","Cha"]}}},"proficiencies":{"classDC":2,"perception":2,"fortitude":4,"reflex":2,"will":4,"heavy":2,"medium":2,"light":2,"unarmored":2,"advanced":0,"martial":2,"simple":2,"unarmed":2,"castingArcane":0,"castingDivine":2,"castingOccult":0,"castingPrimal":0,"acrobatics":2,"arcana":0,"athletics":4,"crafting":0,"deception":0,"diplomacy":0,"intimidation":2,"medicine":0,"nature":0,"occultism":0,"performance":0,"religion":2,"society":0,"stealth":0,"survival":2,"thievery":0},"mods":{},"feats":[["Shield Block",null,"Awarded Feat",1],["Diehard",null,"Awarded Feat",1],["Intimidating Glare",null,"Awarded Feat",1],["Deity's Domain",null,"Class Feat",1,"Champion Feat 1","standardChoice",null],["Hold-Scarred Orc",null,"Heritage",1,"Heritage Feat","standardChoice",null],["Orc Ferocity",null,"Ancestry Feat",1,"Orc Feat 1","standardChoice",null],["Hefty Hauler",null,"Skill Feat",2,"Skill Feat 2","standardChoice",null],["Shining Oath",null,"Class Feat",2,"Champion Feat 2","standardChoice",null],["Toughness",null,"General Feat",3,"General Feat 3","standardChoice",null],["Titan Wrestler",null,"Skill Feat",4,"Skill Feat 4","standardChoice",null],["Aura of Courage",null,"Class Feat",4,"Champion Feat 4","standardChoice",null]],"specials":["Retributive Strike","Deific Weapon","Darkvision","Champion's Code","Tenets of Good","Devotion Spells","Domain: Soul","Paladin [Lawful Good]","Divine Ally (Shield)","Hold-Scarred Orc"],"lores":[["Warfare",2]],"equipmentContainers":{"d4607a18-10ca-43ba-bd5a-16d6aabc31c4":{"containerName":"Backpack","bagOfHolding":false,"backpack":true}},"equipment":[["Backpack",1,"Invested"],["Bedroll",1,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Chalk",10,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Flint and Steel",1,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Rope",1,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Rations",2,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Torch",5,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Waterskin",1,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Soap",1,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Grappling Hook",1,"d4607a18-10ca-43ba-bd5a-16d6aabc31c4","Invested"],["Memoir Map",1,"Invested"],["Healing Potion (Lesser)",1,"Invested"],["Healing Potion (Minor)",1,"Invested"],["Bolts",10,"Invested"],["Eye Slash",1,"Invested"]],"specificProficiencies":{"trained":[],"expert":[],"master":[],"legendary":[]},"weapons":[{"name":"Glaive","qty":1,"prof":"martial","die":"d8","pot":0,"str":"","mat":null,"display":"Glaive","runes":[],"damageType":"S","attack":10,"damageBonus":4,"extraDamage":[],"increasedDice":false,"isInventor":false},{"name":"Battle Axe","qty":1,"prof":"martial","die":"d8","pot":1,"str":"","mat":null,"display":"+1 Battle Axe","runes":[],"damageType":"S","attack":11,"damageBonus":4,"extraDamage":[],"increasedDice":false,"isInventor":false},{"name":"Javelin","qty":5,"prof":"simple","die":"d6","pot":0,"str":"","mat":null,"display":"Javelin (x5)","runes":[],"damageType":"P","attack":8,"damageBonus":4,"extraDamage":[],"increasedDice":false,"isInventor":false}],"money":{"cp":0,"sp":6,"gp":3,"pp":0},"armor":[{"name":"Full Plate","qty":1,"prof":"heavy","pot":0,"res":"","mat":null,"display":"Full Plate","worn":true,"runes":[]},{"name":"Steel Shield","qty":1,"prof":"shield","pot":0,"res":"","mat":null,"display":"","worn":true,"runes":[]}],"spellCasters":[],"focusPoints":2,"focus":{"divine":{"cha":{"abilityBonus":1,"proficiency":2,"itemBonus":0,"focusCantrips":[],"focusSpells":["Eject Soul","Lay on Hands"]}}},"formula":[],"acTotal":{"acProfBonus":6,"acAbilityBonus":0,"acItemBonus":6,"acTotal":22,"shieldBonus":"2"},"pets":[],"familiars":[]}}
 - Details of any errors in the browser console (press F12).

**Comment by Grizzally** [comment]
Never mind, I do believe this is intentional as Pharasma isn't avaiable to Holy War (???) so, I will Alter this in my game and have it Homebrewed.

**Comment by MrPrimate** [comment]
> Never mind, I do believe this is intentional as Pharasma isn't avaiable to Holy War (???) so, I will Alter this in my game and have it Homebrewed.
> 

This is correct, I do believe they are going to try and introduce better support for custom deities later this year, so fingers crossed!
